http_2.rflx

```diff
@@ -78,39 +78,26 @@ package HTTP_2 is
 
    type Frame is
       message
-         Payload_Length : Payload_Length
-            then Frame_Type
-               if Message'Size = (Payload_Length + 9) * 8; -- ISSUE: Componolit/RecordFlux#517
+         Payload_Length : Payload_Length;
          Frame_Type : Frame_Type;
-         Flag_End_Stream_ACK : Boolean
-            then Bit_2_Unused
-               if ((Frame_Type = SETTINGS and Payload_Length = 6)
-                  or (Frame_Type = PING and Payload_Length = 8)
-                  or Frame_Type = DATA
-                  or Frame_Type = HEADERS
-                  or ((Frame_Type = PUSH_PROMISE
-                     or Frame_Type = CONTINUATION
-                     or Frame_Type = GOAWAY
-                     or Frame_Type = RST_STREAM
-                     or (Frame_Type = WINDOW_UPDATE and Payload_Length = 4)
-                     or (Frame_Type = PRIORITY and Payload_Length = 5)
-                  ) and Flag_End_Stream_ACK = False));
-         Bit_2_Unused : Zero_Bit;
-         Flag_End_Headers : Boolean
-            then Flag_Padded
+         Bit_7_Unused : Zero_Bit;
+         Bit_6_Unused : Zero_Bit;
+         Flag_Priority : Boolean
```

Comment:
I would opt for `Flag_Bit_X_Unused` to avoid confusion with the RFC naming. 
The RFC defines flags e.g. like this: `PADDED (0x8):  When set, bit 3 indicates...`.

Reply:
Maybe it would be even better to rename the used flag bits to e.g. `Flag_Bit_3_Padded` additionally.

Reply:
> I would opt for `Flag_Bit_X_Unused` to avoid confusion with the RFC naming.

I'm fine with that.

> Maybe it would be even better to rename the used flag bits to e.g. `Flag_Bit_3_Padded` additionally.

I think `Flag_Padded` is sufficient. Except for the definition of the flag, the RFC just uses the name (e.g., `This field is only present if the PADDED flag is set.`).

Reply:
Ok. changed the names of the unused flags in 94f09b1